### PR TITLE
Change to nf-core tools 2.2 style (Large PR, and we have a ways to go, so merging immediately)

### PR DESCRIPTION
**Pull request by chasemc**
https://github.com/nf-core/tools/blob/e563811adb292b07ec933a52611e937ce9ef5481/CHANGELOG.md?plain=1#L3

### CODE REVIEW

File: lib/WorkflowMain.groovy [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //
-// This file holds several functions specific to the main.nf workflow in the nf-core/benchmark pipeline
+// This file holds several functions specific to the main.nf workflow in the nf-core/socialgene pipeline
```
Comment (WiscEvan) [inline lib/WorkflowMain.groovy:2]: TODO: Replace `socialgene` with `metaBenchmarks`

File: lib/WorkflowMetabenchmarks.groovy [hunk 2]
```diff
@@ -1,19 +1,15 @@
 //
-// This file holds several functions specific to the workflow/benchmark.nf in the nf-core/benchmark pipeline
+// This file holds several functions specific to the workflow/socialgene.nf in the nf-core/socialgene pipeline
```
Comment (WiscEvan) [inline lib/WorkflowMetabenchmarks.groovy:2]: TODO: Replace mentions of `Socialgene` with `metaBenchmarks`

File: lib/WorkflowMain.groovy (hunk 1 above)
Comment (chasemc) [reply lib/WorkflowMain.groovy:2]: https://github.com/KwanLab/metaBenchmarks/pull/11

File: lib/WorkflowMetabenchmarks.groovy (hunk 2 above)
Comment (chasemc) [reply lib/WorkflowMetabenchmarks.groovy:2]: https://github.com/KwanLab/metaBenchmarks/pull/11

**Pull request merged**